.travis.yml

```diff
@@ -7,7 +7,11 @@ sudo: false
 script:
   - cargo build --verbose
   - cargo test --verbose
-  - [ "$TRAVIS_RUST_VERSION" = "1.0" ] || cargo test --verbose --features no_std
+  - if [ "$TRAVIS_RUST_VERSION" = "nightly" ]; then
+       cargo test --verbose --features no_std
```

Comment:
Yeah for now we try to get the most coverage possible (e.g. keep this back to 1.0 while possible)